### DISCUSSION TITLE
Hibernate Query Language (HQL) Dialect

### DISCUSSION THREAD

**Discussion by matanitah** [reata/sqllineage#454] [General]
Is there a way for me to configure the dialect to hql? I don't see a list of available sql dialects to pass in to the Lineage Runner.
Any help would be appreciated!

**Comment by reata** [comment]
The following command gives you the available dialects:

```shell
$ sqllineage --dialects
non-validating
ansi
athena
bigquery
clickhouse
databricks
db2
exasol
hive
materialize
mysql
oracle
postgres
redshift
snowflake
soql
sparksql
sqlite
teradata
tsql
```

I hardly think [Hibernate Query Language](https://docs.jboss.org/hibernate/orm/3.5/reference/en/html/queryhql.html) is a SQL dialect.

Feel free to try non-validating, it might give you some results, although we're looking at deprecating this dialect in the future.
